### DISCUSSION TITLE
关于父分类和子分类的问题

### DISCUSSION THREAD

**Discussion by Macxx-could** [halo-dev/halo#4608] [Q&A] [answered]
假设我现在存在一个父级分类前端，然后下面有一个子级分类vue，然后我有ABC3篇文章，分类选的都是vue，那我在点击查看前端分类的时候，为什么不会把ABC3篇文章都显示出来呀？但是在统计前端分类的文章数量的时候，是有把vue分类文章的数量也算进去的。
![image](https://github.com/halo-dev/halo/assets/82356512/5d68abbf-6315-45f8-af6c-d761c25c740d)
![image](https://github.com/halo-dev/halo/assets/82356512/a551d4a3-a0df-4b3c-9b4b-6b2f8a48d9b1)

**Comment by JohnNiang** [comment, marked as answer]
主题端分类的设计类似于操作系统的”文件夹“的树形设计。如果子节点没有文章则不会显示。
